config/puma.rb

```diff
@@ -15,6 +16,14 @@
 #
 environment ENV.fetch("RAILS_ENV") { "development" }
 
+if ENV['RAILS_ENV'] == 'development'
+  options = {
+    :port => 3000
```

Comment:
If you look higher up in the file, Puma is using the port sourced from the `PORT` env variable, with a fall back on port 3000. Please update this line to use the same port so that ngrok still works even if I configure a different port.

```suggestion
    :port => ENV.fetch("PORT") { 3000 }
```

Reply:
I have refactored this section